tedana/workflows/tedana.py

```diff
@@ -190,10 +171,38 @@ def _get_parser():
                           action='store_true',
                           default=False)
     parser._action_groups.append(optional)
+
+    rerungrp = parser.add_argument_group('arguments for rerunning the workflow')
+    rerungrp.add_argument('--t2smap',
+                          dest='t2smap',
+                          metavar='FILE',
+                          type=lambda x: is_valid_file(parser, x),
+                          help=('Precalculated T2* map'),
```

Comment:
Perhaps this should specify that the T2* map has the same voxel dimensions as the inputted multiecho data?

Reply:
Should be resolved now.